metadata.mk

```diff
@@ -22,14 +22,11 @@ UBI_VERSION=8.10
 
 # Configuration for Semaphore/Github integration.
 ORGANIZATION = projectcalico
+GIT_REPO_NAME = calico
```

Comment:
you might want to update the release tool as well so the env vars are aligned: https://github.com/projectcalico/calico/blob/master/release/cmd/flags.go#L57-L62
from 
```go
	repoFlag = &cli.StringFlag{
		Name:    "repo",
		Usage:   "The GitHub repository to use for the release",
		EnvVars: []string{"GIT_REPO"},
		Value:   utils.Calico,
	}
```
to
```go
	repoFlag = &cli.StringFlag{
		Name:    "repo",
		Usage:   "The GitHub repository to use for the release",
		EnvVars: []string{"GIT_REPO"_NAME},
		Value:   utils.Calico,
	}
```

Reply:
I'll sync up with your name; I think there are other env vars that use that pattern.

Reply:
sounds good!